src/main/java/com/intel/compression/spark/IntelCompressionCodecBlockOutputStream.java

```diff
@@ -21,11 +21,15 @@
 import java.io.OutputStream;
 import java.io.FilterOutputStream;
 import java.nio.ByteBuffer;
+
+import com.intel.compression.util.Platform;
+import io.netty.buffer.ByteBuf;
```

Comment:
do we need add netty dependency in pom?

Reply:
netty dependency should included in the ones of Spark.